domain/consensus/datastructures/consensusstatestore/tips.go

```diff
@@ -72,3 +56,21 @@ func (css *consensusStateStore) deserializeTips(tipsBytes []byte) ([]*externalap
 
 	return serialization.DBTipsToTips(dbTips)
 }
+
+func (csss *consensusStateStagingShard) commitTips(dbTx model.DBTransaction) error {
```

Comment:
Hmmm... this and commitVirtualUTXODiff were deliberately left in their respective files, since consensus-state-store is split to file according to themes, not types.

Thus I felt it makes more sense to leave them here, if you disagree lmk, I don't mind moving it to consensus_state_staging_shard.go too much